axelrod/load_data_.py

```diff
@@ -0,0 +1,45 @@
+import pkg_resources
```

Comment:
Not sure how to better test these than simply loading the data and testing the strategies. I added a few integrity checks in ANN and LookerUp to make sure the data is of the expected length.